easybuild/easyconfigs/c/Cantera/Cantera-2.6.0-foss-2022a.eb

```diff
@@ -15,9 +15,13 @@ checksums = ['7273622ea76a53373cee820f939613b15eea3dd83db6e1b127c5ed043f77dc5b']
 
 dependencies = [
     ('Python', '3.10.4'),
-    ('SciPy-bundle', '2022.05'),
```

Comment:
From your PR commit message, it was clear why the `SUNDIALS` version changed. Can you comment on why `SciPy-bundle` was removed as dependency, and why `h5py`-`PyQt5` were added?

Reply:
https://www.cantera.org/compiling/installation-reqs.html#sec-installation-reqs

H5 is an optional dependency requested by final users. SciPy-bundle still comes from dependency . PyQt5 for plot charts `matplotlib.use('QtAgg')` for instance.

Reply:
> H5 is an optional dependency requested by final users.

Makes sense, it add optional functionality. That's great.

> SciPy-bundle still comes from dependency

Normally I would say "If X is a dependency of Y, it should be listed, even if X is also a dependency of Z, which is a dependency of Y". However, from your link, I see that `scipy` is optional, and only needed for some examples. So, removing it is fine by me.

> PyQt5 for plot charts matplotlib.use('QtAgg') for instance

Well, according to that argument, it should be a dependency for `matplotlib` :) (which it also isn't, but that's another discussion), but not for Cantera. If users want to use the `QtAgg` backend, they can load the `PyQt5` module themselves. I guess this is one of the ambiguous cases of how to deal with optional dependencies in EasyBuild. One reason to be a bit conservative here is that Qt5 is big, takes ages to build, and build problems are quite common. Thus, you don't really want to include it as a hard dependency unless really needed. By not including it as a dependency here (or in matplotlib), Cantera (and matplotlib) can be installed and used, even on systems where building Qt5 proves problematic. Summarizing: please remove it from the dependency list :)

Reply:
I removed PyQt5 as dependencies. Tnx for your support :)

Reply:
Very welcome :)